BuildModule.ps1

```diff
@@ -0,0 +1,152 @@
+#Requires -RunAsAdministrator
+
+param (
+    [Parameter(Mandatory=$false)][string]$msbuild,
+    [Parameter(Mandatory=$false)][string]$framework
+)
+
+function FindMSBuild {
```

Comment:
Change this to the BuildModule.ps1 used by the Decisions.FHIR module

Reply:
Updated